### ISSUE TITLE
Update citation for Quarto v1.4

### ISSUE THREAD

**Issue by coatless** [quarto-dev/quarto-cli#8395]
The latest citation information refers to Quarto v1.2. 

https://github.com/quarto-dev/quarto-cli/blob/df1eda553a8407e4e1c575edd55310111049028d/CITATION.cff#L20-L22

Can the citation information/zenodo be updated when Quarto v1.4 is released? And, on each future full release of Quarto?

**Comment by cderv** [comment]
I updated the citation file for date in the repo. 36922928168fa352940767e8a2cde12ab4d2b7f8 

This will make the Cite this repository button have the right version and date. 

Though DOI is not updated yet - I must acknowledge, I don't know yet how to update. 

After I learn all this, I'll automate with our stable release process in our CI. So I keep this open.

**Comment by mcanouil** [comment]
This seems to be reolved.

**Comment by coatless** [comment]
@mcanouil I think this was a manual process, c.f. @.cscheid 

https://github.com/quarto-dev/quarto-cli/commits/main/CITATION.cff

**Comment by mcanouil** [comment]
Hum, so manual update the last three releases.

FWIW, I've been doing this automatic update for my Quarto extensions (https://github.com/mcanouil/quarto-workflows/blob/main/.github/workflows/release-extension.yml)

**Comment by cderv** [comment]
yes this still need to be automated. The missing part if the DOI which is completely out of date. 
https://zenodo.org/records/5960048

I did not take time to see how I can update those informations there.

**Comment by mcanouil** [comment]
I'll try to take a look.

To summarise, the file should be updated with:
- date
- DOI
- major.minor version


The DOI is the tricky part, because it needs a release/tag to be generated, so the CFF file can only be truly up to date after that not before release.

**Comment by mcanouil** [comment]
Alternatively, Zenodo provides a global DOI rather than one tied up to a version.
Using that DOI would ensure the DOI is always correct because the "global" DOI is redirected to the latest one. This would also simplify substantially the automatisation of that task.

**Comment by cderv** [comment]
Yep, it was not created this way by JJ at the time. I still need to see what can be done. So still WIP.

**Comment by mcanouil** [comment]
Oh, it's not integrated. The Zenodo DOI was done manually, so it only has one record for v1.0. (https://zenodo.org/records/5960048)

So, it's not really only about the automatisation but about the use of Zenodo for Quarto releases which is more a matter of discussion than implementation.

(I can't find again the documentation about the DOI global/version from Zenodo)

**Comment by cderv** [comment]
yes exactly.
